test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -347,6 +353,63 @@ def _verify_symmetric_xnnpack_qat_graph_helper(
         self.assertTrue("bn_running_var" in bn_running_var_node.target)
         self.assertEqual(eps, 1e-5)
 
+        # Optionally verify convert
+        if not verify_convert:
+            return
+        m = convert_pt2e(m)
+        m(*example_inputs)
+
+        # Verify: conv [- relu] - q - dq - output
+        output_node = list(m.graph.nodes)[-1]
+        output_dq_node = output_node.args[0][0]
+        output_q_node = output_dq_node.args[0]
+        if has_relu:
+            relu_node = output_q_node.args[0]
+            self.assertEqual(relu_node.target, torch.ops.aten.relu.default)
+            conv_node = relu_node.args[0]
+        else:
+            conv_node = output_q_node.args[0]
+        self.assertEqual(
+            output_dq_node.target,
+            torch.ops.quantized_decomposed.dequantize_per_tensor.default,
+        )
+        self.assertEqual(
+            output_q_node.target,
+            torch.ops.quantized_decomposed.quantize_per_tensor.default,
+        )
+        self.assertTrue(_is_conv_node(conv_node))
+        self.assertEqual(list(output_dq_node.args[-3:]), [-128, 127, torch.int8])
+        self.assertEqual(list(output_q_node.args[-3:]), [-128, 127, torch.int8])
+
+        # Verify: conv_weight - weight_dq - conv
```

Comment:
for these can we use existing self.checkGraphModuleNodes or something like that to check?

Reply:
Ok let me try. I think we can do this for the convert graph since it's simpler. For the prepare graph we probably still need the existing code